webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -5650,16 +5650,13 @@ var userOverride = {
             _this.sendMessage = function (messagePxy, file, callback) {
                 var key;
                 var message;
+                var lastMessageBeforeSend = $applozic("#mck-message-cell .mck-message-inner div[name='message']:last-child");
+
                 kmWidgetEvents.eventTracking(eventMapping.onMessageSent);
                 if (
-                    Kommunicate.internetStatus &&
-                    $applozic(
-                        "#mck-message-cell .mck-message-inner div[name='message']:last-child"
-                    ).data('msgkey') !== undefined
+                    Kommunicate.internetStatus && lastMessageBeforeSend.data('msgkey') !== undefined
```

Comment:
is null check required for lastMessageBeforeSend?

Reply:
if you check `$applozic("#mck-message-cell .mck-message-inner div[name='message']:last-child"`

then we haven't added any null check, i just assign this declaration to a variable as I am using this 3 times although let me check (If there are no message from bot/agent and user is trying to send the message)

Reply:
it is working fine, if there are no welcome message from bot/agent